src/views/StandaloneDirectorsFiling.vue

```diff
@@ -767,60 +794,37 @@ export default {
             const changeOfDirectors = filing.changeOfDirectors
             if (changeOfDirectors) {
               if (changeOfDirectors.directors && changeOfDirectors.directors.length > 0) {
-                if (this.$refs.directorsList && this.$refs.directorsList.setAllDirectors) {
+                if (this.$refs.directorsList.setAllDirectors) {
                   this.$refs.directorsList.setAllDirectors(changeOfDirectors.directors)
                 }
-
-                // add filing code for paid changes
-                if (changeOfDirectors.directors.filter(
```

Comment:
Unneeded code. It causes duplicate updates. The Directos component already emits events to do this.